src/app/hooks/useOptimizelyVariation/index.test.jsx

```diff
@@ -1,4 +1,4 @@
-import optimizelyReactSdk from '@optimizely/react-sdk';
+import * as optimizelyReactSdk from '@optimizely/react-sdk';
```

Comment:
Ignore this change - thought I was fixing the unit test but actually this broke it. (I forgot to run yarn before running jest)